src/components/charts/WorldMap.js

```diff
@@ -0,0 +1,116 @@
+import React from "react"
+import _ from "lodash"
```

Comment:
we do not use lodash directly, use "lodash.find" instead (need to yarn add separately)